### ISSUE TITLE
This looks like a sprite, not an SDF?

### ISSUE THREAD

**Issue by jamesford42** [Chlumsky/msdfgen#214]
Are we certain that the output for regular-sdf mode is actually working? 

Because it seems to me like its rasterizing a normal sprite. The black areas are perfectly black, and the white areas are perfectly white. 

Using this command:
```msdfgen sdf -svg club.svg -dimensions 512 512 -autoframe -o club.png```
And this input file (club.svg):
![club](https://github.com/user-attachments/assets/afd86eca-6db3-42a7-9982-a855d5a6ae6f)
It outputs this:
![club](https://github.com/user-attachments/assets/a7c7f795-d958-4a51-b805-7149cb7d5fa0)

.. which doesn't look like a signed distance field, to me?

**Comment by Chlumsky** [comment]
Yes, we are certain. The default distance range is 2 pixels, you have to set it.
